Dockerfile

```diff
@@ -0,0 +1,45 @@
+
+FROM openjdk:8-jre-alpine
+
+# Expose ports
+EXPOSE 80
+
+# Add tags
+LABEL \
+  maintainer="The Petals team" \
+  contributors="Pierre Souquet (Linagora), Vincent Zurczak (Linagora)" \
+  github="https://github.com/petalslink"
+
+# Build arguments
+ARG CLI_VERSION="LATEST"
+ARG MAVEN_POLICY="releases"
+ARG BASE_URL="https://repository.ow2.org/nexus/service/local/artifact/maven/redirect"
+
+# Environment variables
+ENV ENV_CLI_HOST="localhost"
```

Comment:
All these ENV_ variables are useless.
The docker-wrapper script should verify they exist. If they do not, then it should set the default version. See https://github.com/roboconf/roboconf-dockerfile/blob/master/start.sh